bootcamp/tutorials/integration/llamaindex/llamaindex_milvus_hybrid_search.ipynb

```diff
@@ -0,0 +1,586 @@
+{
```

Comment:
<p>This notebook demonstrates how to use Milvus for hybrid search in&nbsp;<a href="https://www.llamaindex.ai/" target="_blank">LlamaIndex</a>&nbsp;RAG pipelines. We'll begin with the recommended default hybrid search (semantic + BM25) and then explore other alternative sparse embedding methods and customization of hybrid reranker.</p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1527/discussion/#comment-2044679197'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>